.github/workflows/_linux-build.yml

```diff
@@ -92,6 +92,15 @@ jobs:
         uses: pytorch/test-infra/.github/actions/calculate-docker-image@main
         with:
           docker-image-name: ${{ inputs.docker-image-name }}
+      - name: Use following to pull public copy of the image
```

Comment:
Should this be included as a step on `_linux_test` too?

Reply:
Yes, and alas it can not be reusable workflow yet, as it can not be rebased properly